app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -2,30 +2,68 @@ class DradisDatatable {
   constructor(tableElement) {
     this.$table = $(tableElement);
     this.dataTable = null;
+    this.tableHeaders = Array.from(this.$table[0].querySelectorAll('thead th, thead td'));
```

Comment:
We don't need `td` and `th`, if there's an instance of `thead > td` we need to fix the HTML source to turn that into a `th`